ios/FluentUI/Vnext/Notification/NotificationView_SwiftUI.swift

```diff
@@ -78,6 +96,18 @@ public struct NotificationViewSwiftUI: View, ConfigurableTokenizedControl {
         !state.style.isToast && state.actionButtonAction == nil
     }
 
+    private func presentAnimated() {
+        withAnimation(.spring(response: tokens.style.animationDurationForShow, dampingFraction: tokens.style.animationDampingRatio, blendDuration: 0)) {
+            yOffsetFromBottom = 0
+        }
+    }
+
+    private func dismissAnimated() {
+        withAnimation(.spring(response: tokens.style.animationDurationForHide, dampingFraction: tokens.style.animationDampingRatio, blendDuration: 0)) {
```

Comment:
If this animation is going to be the same in multiple places, it could be a computed property to prevent duplication:

```
    private var presentationAnimation: Animation {
        .spring(response: tokens.style.animationDurationForShow,
                dampingFraction: tokens.style.animationDampingRatio,
                blendDuration: 0)
    }
```